app/assets/stylesheets/pages/_roadmap.scss

```diff
@@ -0,0 +1,108 @@
+
+  $roadmap-bg: #4D4545;
+  $roadmap-color: #007bff;
+  $roadmap-font-stack: 'Lato', sans-serif;
+
+.roadmap {
+
+  font: 100% $roadmap-font-stack;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/font

just wondering what the 100% would apply to